inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/BusinessController.java

```diff
@@ -86,7 +86,8 @@
     @OperationLog(operation = OperationType.DELETE)
     @ApiImplicitParam(name = "groupId", value = "Business group id", dataTypeClass = String.class, required = true)
     public Response<Boolean> delete(@PathVariable String groupId) {
-        return Response.success(businessService.delete(groupId, LoginUserUtil.getLoginUserDetail().getUserName()));
+        return Response.success(bizProcessOperation.deleteProcess(groupId, 
```

Comment:
It is recommended to distinguish between deleting data and deleting processes.

Reply:
BusinessProcessOperation will invoke BusinessService to logically delete data, I think data changing should all lay up cohensively in BusinessService, and repository class should not be invoked directly by other classes;

Reply:
OK, in `deleteProcess` method, logically delete business info.